config/routes.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
-devise_for :users
+  devise_for :users
 
-root 'welcome#index'
+ root "welcome#index"
```

Comment:
Inconsistent indentation detected.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.